recipes/ri/makefile.patch

```diff
@@ -0,0 +1,19 @@
+--- makefile    2020-06-03 13:15:56.267331987 +0100
++++ makefile    2020-06-03 13:15:37.639335523 +0100
+@@ -1,6 +1,4 @@
+ INCLUDES= -I ./rilib/ -I ./include/
+-CC=g++
+-CFLAGS=-c -O3
+
+ SOURCES= ri3.cpp
+ OBJECTS=$(SOURCES:.cpp=.o)
+@@ -10,7 +8,7 @@
+ all:    $(SOURCES) $(EXECUTABLE)
+
+ $(EXECUTABLE): $(OBJECTS)
+-    $(CC) $(OBJECTS) -o $@
++    $(CC) $(CFLAGS) $(OBJECTS) -o $@
```

Comment:
This needs to be tabs.
```suggestion
-	$(CC) $(OBJECTS) -o $@
+	$(CC) $(CFLAGS) $(OBJECTS) -o $@
```

Reply:
Yes - I was just applying the change. But you are quicker! 😄